adhocracy-plus/assets/scss/_layout.scss

```diff
@@ -9,7 +9,7 @@
     @supports (display: grid) {
         display: grid;
         grid-column-gap: _grid-get("gutter", $settings);
-        grid-template-columns: repeat($n, 1fr);
+        grid-template-columns: repeat($n, minmax(0, 1fr));
```

Comment:
this is a very fundamental change to the grid. I had the issue described in the following link, which made one column wider than the other two.

Here the excerpt that is relevant:

>The common answer of repeat(3, 1fr) is not quite correct.
>
>This is because 1fr is about the distribution of available(!) space. This breaks as soon as the content becomes >bigger than the track size. By default, it does not overflow and adjust the column width accordingly. That's why >not all 1fr are guaranteed to be of equal width. 1fr is actually rather just a shorthand for minmax(auto, 1fr).
>
>If you really need the columns to be the exact same width you should use:

>` grid-template-columns: repeat(3, minmax(0, 1fr));`

>minmax(0, 1fr) allows the grid tracks to be as small as 0 but as large as 1fr, creating columns that will stay >equal. But, be aware that this will cause overflows if the content is bigger than the column or cannot be >wrapped.

The overflows should be handled? what do you think @phillimorland ?

link to [stackoverflow answer](https://stackoverflow.com/questions/47601564/equal-width-columns-in-css-grid):